src/Components/SearchByInsurance/SearchByInsurance.js

```diff
@@ -8,16 +8,18 @@ const SearchByInsurance = ({ providerList, setFilteredProviderList }) => {
   const handleSearch = (event) => {
     event.preventDefault()
     const searchResults = filterProvider()
-    if (searchResults.length) {
+
+    if (searchResults?.length) {
       setFilteredProviderList(searchResults)
     } else {
       setError(`Sorry! We don't have any results for ${searchInput}`)
     }
     setSearchInput('')
   }
 
+
   const filterProvider = () => {
-    const filteredProviders = providerList.filter(provider => {
+    const filteredProviders = providerList?.filter(provider => {
```

Comment:
Neat shorthand! I learned something about optional chaining today

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Here is the mdn on the short hand.  Optional chaining checks for undefined status.